### PR TITLE
Switching tabs position to center Discover

### PR DESCRIPTION
**Pull request by yaelirub**
This PR switches the position of the 5 item tab bar to center Discover. The new order is Podcasts, Filters, Discover, Up Next, Profile. This order seems to reflect access levels on Android per data collected in recent release. 

Simulator Screenshot - iPhone 15 Pro - 2024-06-04 at 19 39 07

- [ ] I have considered if this change warrants user-facing release notes and have added them to `CHANGELOG.md` if necessary.
- [ ] I have considered adding unit tests for my changes.
- [ ] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/Automattic/pocket-casts-ios/milestone/50">7.65 ❄️</a>. The due date for this milestone has already passed.<br />
Please assign it to a milestone with a later deadline or check whether the release for this milestone has already been finished.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: .xcode.env [hunk 1]
```diff
@@ -0,0 +1 @@
+export NODE_BINARY=
```
Comment (SergioEstevao) [inline .xcode.env]: Was this change intentional?

**Review by SergioEstevao — APPROVED** [review]
Working great just needs to cleanup the change on the `.xcode.env` file.

File: .xcode.env (hunk 1 above)
Comment (yaelirub) [reply .xcode.env]: It was not. Will clean

Comment (yaelirub) [reply .xcode.env]: reverted
